libs/gamify/data/src/lib/services/applications.service.ts

```diff
@@ -1,15 +1,16 @@
+import { ApplicationModel, ApplicationUserModel, CreateApplicationInput, UpdateApplicationInput } from '@gamify/application';
 import { Role } from '@gamify/core';
 import { DataService } from '@gamify/data';
-import { UsersService } from '@gamify/users';
-import { Injectable } from '@nestjs/common';
-import { CreateApplicationInput } from './dto/create-application.input';
-import { UpdateApplicationInput } from './dto/update-application.input';
-import { ApplicationModel, ApplicationUserModel } from './models';
+import { forwardRef, Inject, Injectable } from '@nestjs/common';
+import { UsersService } from './users.service';
 
 @Injectable()
 export class ApplicationsService {
 
-    constructor(private readonly data: DataService, private readonly usersService: UsersService) {}
+    constructor(
+        private readonly data: DataService, 
+        @Inject(forwardRef(() => UsersService)) private readonly usersService: UsersService
```

Comment:
Is this (still) necessary? (forwardRef)

Reply:
It is not, removed it in the next commit